PocketKit/Sources/PocketKit/Home/HomeViewController.swift

```diff
@@ -216,17 +216,22 @@ extension HomeViewController {
 
     func viewForSupplementaryElement(ofKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
         switch kind {
-        case SlateHeaderView.kind:
-            let header: SlateHeaderView = collectionView.dequeueReusableView(forSupplementaryViewOfKind: kind, for: indexPath)
+        case SectionHeaderView.kind:
+            let header: SectionHeaderView = collectionView.dequeueReusableView(forSupplementaryViewOfKind: kind, for: indexPath)
             let section = dataSource.sectionIdentifier(for: indexPath.section)
-            
             switch section {
             case .recentSaves:
-                let presenter = SlateHeaderPresenter(name: "Recent Saves")
-                header.attributedHeaderText = presenter.attributedHeaderText
+                let presenter = SectionHeaderPresenter(name: "Recent Saves", buttonTitle: "My List")
+                header.configure(model: .init(attributedHeaderText: presenter.attributedHeaderText,
```

Comment:
Idea: consolidate the presenter and header model into one. Then, when we configure the header with a model, we can initialize the model with the slate as well as the action, rather than its individual properties (currently set at initialization). To me, the presenter kind of feels like an extra type for sake of extracting computed properties.

Reply:
updated!